src/main/java/com/softserve/kickscootertrip/model/dto/TripDto.java

```diff
@@ -0,0 +1,20 @@
+package com.softserve.kickscootertrip.model.dto;
+
+import lombok.Data;
+
+import java.util.Date;
+import java.util.UUID;
+
+@Data
+public class TripDto {
+    private UUID tripId;
+    private UUID userId;
+    private UUID scooterId;
+    private Date start;
```

Comment:
Maybe use LocalDateTime ? TripTime will be difference beetween start and finish.